pkg/installer/install

```diff
@@ -99,7 +99,7 @@ root_dev() {
    local MAJOR
    local MINOR
    local DEV
-   if [ -L /dev/root ] ; then
+   if [ -L /dev/root ] || [ -b /dev/root ] ; then
```

Comment:
Could we make it even easier with 
```suggestion
   if [ -e /dev/root ] ; then
```

Reply:
-e would make the if pass with any kind of file, even a regular file, which is not what we are looking for, I prefer to let as it is....

Reply:
Can it really happen that `/dev/root` exists but is not a block device or a link to a block device?

Reply:
actually I was wrong.... `-e` won't pass and it will fail since the file is not a regular file, but either a symbolic link or a special block file device, so we can't use `-e` as you suggested, it must be the checking provided in the PR....

Reply:
Argh, sorry, you're right...